code/modules/religion/sects/shadow_sect.dm

```diff
@@ -60,6 +64,88 @@
 	icon_state = "shadow-obelisk"
 	anchored = FALSE
 	break_message = "<span class='warning'>The Obelisk crumbles before you!</span>"
+	var/heal_delay = 20
+	var/last_heal = 0
+	var/spread_delay = 45
+	var/last_spread = 0
+
+/obj/structure/destructible/religion/shadow_obelisk/Initialize(mapload)
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	. = ..()
+	if(!sect.faithful_used)
+		return
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/destructible/religion/shadow_obelisk/LateInitialize()
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	. = ..()
+	if(!sect.faithful_used)
+		return
+	START_PROCESSING(SSobj, src)
```

Comment:
This logic is very strange, why are you doing it this way?

Reply:
Didn't really know how to sort it tbh😅 if you've got suggestions I'll take them

Reply:
Looking over it, I think I wanted to avoid having the obelisks run `process()` if `faithful_used` wasnt true, but I *think* I can just put START_PROCESSING in a seperate proc relating to the rite, maybe?

Reply:
sorted